lib/model.js

```diff
@@ -5,6 +5,8 @@ var assert = require('assert');
 var RemoteObjects = require('strong-remoting');
 var SharedClass = require('strong-remoting').SharedClass;
 var extend = require('util')._extend;
+var normalizeInclude = require('loopback-datasource-juggler/lib/include').normalizeInclude;
```

Comment:
I think it's a bad practice to load deep internals of a dependency. Can you modify juggler to export `normalizeInclude` in such way, that you can load it here via `require('loopback-datasource-juggler').normalizeInclude` or something similar?